.github/workflows/build.yml

```diff
@@ -5,6 +5,17 @@ on:
   push:
     branches: [ main, develop ]
 
+#Custom gradle.properties file only for CI/CD
+env:
+  CI_GRADLE_PROPERTIES: |
+    #org.gradle.jvmargs=-Xmx2g -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8 -XX:+UseParallelGC
+    org.gradle.jvmargs=-Xmx2g
+    org.gradle.vfs.watch=true
+    android.enableJetifier=true
+    android.useAndroidX=true
```

Comment:
It strange to add again the value which are already in 'gradle.properties'. See my other remark to try to avoid that.